test/system/070-build.bats

```diff
@@ -345,7 +345,7 @@ EOF
     # all commands after 'podman build' would silently be ignored.
     # In the test below, prior to #8092, the 'sed' would not get
     # any input, and we would never see $random3 in the output.
-    # And, we use 'sed' to massage $random3 juuuuust on the remote
+    # And, we use 'sed' to massage $random3 just on the remote
```

Comment:
I understand this isn't a typical typo, but this repo doesn't have many items like this, so I'm treating it like one.